test/dlc_tests/sanity/test_torchvision.py

```diff
@@ -25,7 +25,7 @@ def test_torchvision_nms_training(pytorch_training):
 
 
 @pytest.mark.model("N/A")
-def test_torchvision_nms_inference(pytorch_inference):
+def test_torchvision_nms_inference(pytorch_inference, pt16_and_above_only):
```

Comment:
This test applies to all PyTorch versions.

It is meant to test that the custom-torchvision binary has been correctly installed in the DLC.

Reply:
Pytorch team will be re-build the `torchvision` binary again to solve this issue. Will revert the test skip.

Reply:
removed the pt16 condition.